### ISSUE TITLE
"Back to posts" link in the BlogTemplate theme returns 404

### ISSUE THREAD

**Issue by awulkan** [statiqdev/Statiq#581]
Step 1.
Go to this page: https://wyam.io/recipes/blog/themes/preview/blogtemplate/tags/introduction

Step 2.
Click the "Back to posts" link.

You'll get a 404 error since the theme just takes the host string and concatenates it onto the current URL.
`https://wyam.io/recipes/blog/themes/preview/blogtemplate/host.com`

The code doing this is currently:
```cs
<a href="@Context.GetLink(Context.String(Keys.Host))">Back To posts</a>
```
but I think it should be:
```cs
<a href="@Context.GetLink(Context.String(BlogKeys.PostsPath))">Back To posts</a>
```

**Comment by jbrinkman** [comment]
I've submitted a PR with this change.

**Comment by daveaglick** [comment]
Resolved by #583 (the PR referenced the wrong issue number)
